internal/logger/target/http/http.go

```diff
@@ -260,6 +257,7 @@ func (h *Target) send(ctx context.Context, payload []byte, payloadCount int, pay
 
 	resp, err := h.client.Do(req)
 	if err != nil {
+		h.failedMessages.Add(int64(payloadCount))
```

Comment:
I think we can add this in error L240

Reply:
done

Reply:
I think you missed it, probably after a rebase